example/main.tf

```diff
@@ -10,12 +10,29 @@ module "minio" {
   host                            = "127.0.0.1"
   port                            = 9000
   container_image                 = "minio/minio:latest"
-  access_key                      = "minio"
-  secret_key                      = "minio123"
+  access_key                      = var.access_key
+  secret_key                      = var.secret_key
```

Comment:
I initially thought that if variables are not set, then they are 
1. generated automatically
2. stored in the vault
3. injected in nomad job using [consul-template](https://github.com/hashicorp/consul-template#secret)

I guess I had to say that in acceptance criteria under user-story

Reply:
So we can remove `var.acces_key` and `var.secret_key` right?

Reply:
That depends on needs. Right now I can not answer yes or no.

Reply:
But in the future it might be a solution, remove from user responsibility to provide credentials.